contracts/DelFiPrice.sol

```diff
@@ -62,7 +93,16 @@ contract DelFiPrice is OpenOracleView {
         }
 
         uint64[] memory sortedPrices = sort(postedPrices);
-        return sortedPrices[N / 2];
+        // if N is even, get the left and right medians and average them
+        if (N % 2 == 0) {
+            uint64 left = sortedPrices[(N / 2) - 1];
```

Comment:
Is this right? My intuition is that is about be `N/2` and `N/2+1` not `N/2` and `N/2-1`, but that's just intuition.

Reply:
the array is zero-indexed, and N refers to lenght. so for example a 4 item array has indexes 0,1,2,3 and N of 4. We want indices 1 and 2, so N/2 and N/2 -1 is correct. 

might be worth adding a comment since it has come up twice now.